### PR TITLE
dou-fu-zhu-ni-qiu subtitle finished

### PR DESCRIPTION
**Pull request by Hackerry**

### CODE REVIEW

**Comment by immoonancient** [comment]
#581

File: subtitles/wang-gang/200115-dou-fu-zhu-ni-qiu.sbv [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+# 同学们切记这一步必须小心操作
+be careful when performing this step.
+
+0:00:33.360,0:00:36.000
+# 切断泥鳅的头之后用菜刀将尾巴压住
+After the head comes off, use cleaver knife to hold the tail tightly,
+
+0:00:36.140,0:00:39.100
+# 然后用手反方向将腹部的内脏挤出来
+while squeezing the internal organs out.
```
Comment (NoraZheng) [inline subtitles/wang-gang/200115-dou-fu-zhu-ni-qiu.sbv]:
```suggestion
while squeezing out the internal organs.
```

File: subtitles/wang-gang/200115-dou-fu-zhu-ni-qiu.sbv [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+# 准备老豆腐一小块
+Get a small chunk of firm Tofu
+
+0:00:49.265,0:00:52.205
+# 将豆腐从中间切开之后切成厚片备用
+Cut it in half and into thick slices
+
+0:00:56.600,0:00:59.000
+# 准备生姜一小块切成姜片备用
+Get a small piece of ginger and cut into pieces
```
Comment (NoraZheng) [inline subtitles/wang-gang/200115-dou-fu-zhu-ni-qiu.sbv]:
```suggestion
Get a small piece of ginger and cut into slices
```

File: subtitles/wang-gang/200115-dou-fu-zhu-ni-qiu.sbv [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+0:01:00.960,0:01:03.140
+# 准备小葱几根切成葱花备用
+Get several scallion/green onion and shred into pieces
```
Comment (NoraZheng) [inline subtitles/wang-gang/200115-dou-fu-zhu-ni-qiu.sbv]:
```suggestion
Get several scallions/green onions and finely chop them
```

File: subtitles/wang-gang/200115-dou-fu-zhu-ni-qiu.sbv [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+# 然后将处理好的泥鳅下锅煎3分钟
+Then fry loaches for 3 minutes
+
+0:01:26.420,0:01:29.180
+# 同学们煎泥鳅的时候最好用锅盖挡住锅面
+It's better to put a lid over the wok when frying
+
+0:01:29.180,0:01:31.160
+# 这样可以防止热油溅出伤人
+to protect yourself from hot spilling oils.
```
Comment (NoraZheng) [inline subtitles/wang-gang/200115-dou-fu-zhu-ni-qiu.sbv]:
```suggestion
to protect yourself from hot splashing oil.
```

File: subtitles/wang-gang/200115-dou-fu-zhu-ni-qiu.sbv [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+# 再加入切好的豆腐下锅
+And put Tofu into the wok
+
+0:02:03.840,0:02:05.660
+# 然后开大火烧开
+Turn to high heat and heat to a boil
+
+0:02:05.660,0:02:08.100
+# 大火烧开之后转中火煮5分钟
+Then switch to mid-heat and slow boil for 5 minutes
```
Comment (NoraZheng) [inline subtitles/wang-gang/200115-dou-fu-zhu-ni-qiu.sbv]:
```suggestion
Then switch to medium heat and boil for 5 minutes
```

File: subtitles/wang-gang/200115-dou-fu-zhu-ni-qiu.sbv [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+# 锅中加入食用盐一勺
+A spoon of salt
+
+0:02:19.820,0:02:21.120
+# 加入胡椒粉半勺
+And half a spoon of black pepper powder
+
+0:02:24.960,0:02:27.040
+# 调好味之后即可出锅上菜
+When stiring well, pour out the soup and serve
```
Comment (NoraZheng) [inline subtitles/wang-gang/200115-dou-fu-zhu-ni-qiu.sbv]:
```suggestion
After seasoning, pour out the soup and serve
```

**Pull request merged**